app/assets/javascripts/discourse/templates/discovery/categories.hbs

```diff
@@ -22,16 +22,16 @@
                 <div class="category-description">
                   {{{c.description_excerpt}}}
                 </div>
+                {{#if c.category_expanding_class}}
+                    <button class="btn"
+                      {{action "expand" c.category_expanding_class}}
+                      {{bind-attr id="c.category_expanding_class"}}
+                    >
+                        Expand units for this course
+                    </button>
+                {{/if}}
```

Comment:
won't this button be within the very `tr` that it's supposed to be able to show? And therefore itself be hidden?